### PR TITLE
Updating validation sequence for pixel only tracking workflows

### PR DESCRIPTION
**Pull request by sroychow**
#### PR description:
In this PR, the validation modules for pixel digis, clusters and rechits are added to the pixel only tracking  validation sequence. The PR also includes a fix for the cut on ndof for the primary vertex monitor.

#### PR validation:
This has been tested with 11634.0(customized for pixel only tracking), 11634.502.

### CODE REVIEW

File: RecoPixelVertexing/Configuration/python/customizePixelTracksSoAonCPU.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def customizePixelTracksSoAonCPU(process) :
 
   process.reconstruction_step += process.siPixelRecHitHostSoA+process.pixelTrackSoA+process.pixelVertexSoA
 
+
```
Comment (fwyzard) [inline RecoPixelVertexing/Configuration/python/customizePixelTracksSoAonCPU.py:25]:
```suggestion
```

File: Validation/Configuration/python/postValidation_cff.py [hunk 2]
```diff
@@ -73,9 +73,10 @@
 
 postValidation_trackingOnly = cms.Sequence(
       postProcessorTrackSequenceTrackingOnly
-    + postProcessorVertexSequence
+    + postProcessorVertexSequence	
 )
 
+
```
Comment (fwyzard) [inline Validation/Configuration/python/postValidation_cff.py:79]:
```suggestion
    + postProcessorVertexSequence
)
```

**Review by fwyzard — COMMENTED** [review]
Clean up whitespace changes

File: Validation/SiPixelPhase1ConfigV/python/SiPixelPhase1OfflineDQM_sourceV_cff.py [hunk 3]
```diff
@@ -20,3 +20,32 @@
                                             + SiPixelPhase1TrackingParticleAnalyzerV
                                             )
 
+###PixelTracking only configurations for gpu wf###
+#digi
+pixelOnlyDigisAnalyzerV = SiPixelPhase1DigisAnalyzerV.clone()
+#cluster
+pixelOnlyTrackClustersAnalyzerV = SiPixelPhase1TrackClustersAnalyzerV.clone()
+pixelOnlyTrackClustersAnalyzerV.clusters = cms.InputTag('siPixelClustersPreSplitting')
+pixelOnlyTrackClustersAnalyzerV.tracks = cms.InputTag('pixelTracks')
```
Comment (fwyzard) [inline Validation/SiPixelPhase1ConfigV/python/SiPixelPhase1OfflineDQM_sourceV_cff.py:29]:
```suggestion
pixelOnlyTrackClustersAnalyzerV = SiPixelPhase1TrackClustersAnalyzerV.clone(
    clusters = 'siPixelClustersPreSplitting',
    tracks = 'pixelTracks'
)
```

Comment (fwyzard) [reply Validation/SiPixelPhase1ConfigV/python/SiPixelPhase1OfflineDQM_sourceV_cff.py:29]: I have been told to prefer this syntax when making changes to a module after cloning it.

File: Validation/SiPixelPhase1ConfigV/python/SiPixelPhase1OfflineDQM_sourceV_cff.py [hunk 4]
```diff
@@ -20,3 +20,32 @@
...
+
+#rechit analyzer
+pixelOnlyRecHitsAnalyzerV = SiPixelPhase1RecHitsAnalyzerV.clone()
+pixelOnlyRecHitsAnalyzerV.src = cms.InputTag("siPixelRecHitsPreSplitting")
+pixelOnlyRecHitsAnalyzerV.pixelSimLinkSrc = cms.InputTag("simSiPixelDigis")
+pixelOnlyRecHitsAnalyzerV.ROUList = cms.vstring('TrackerHitsPixelBarrelLowTof',
+                                                'TrackerHitsPixelBarrelHighTof',
+                                                'TrackerHitsPixelEndcapLowTof',
+                                                'TrackerHitsPixelEndcapHighTof')
```
Comment (fwyzard) [inline Validation/SiPixelPhase1ConfigV/python/SiPixelPhase1OfflineDQM_sourceV_cff.py:38]:
```suggestion
pixelOnlyRecHitsAnalyzerV = SiPixelPhase1RecHitsAnalyzerV.clone(
    src = 'siPixelRecHitsPreSplitting',
    pixelSimLinkSrc = 'simSiPixelDigis',
    ROUList = ('TrackerHitsPixelBarrelLowTof',
               'TrackerHitsPixelBarrelHighTof',
               'TrackerHitsPixelEndcapLowTof',
               'TrackerHitsPixelEndcapHighTof')
)
```

File: Validation/SiPixelPhase1ConfigV/python/SiPixelPhase1OfflineDQM_sourceV_cff.py [hunk 5]
```diff
@@ -20,3 +20,32 @@
...
+#Hits
+pixelOnlyHitsAnalyzerV = SiPixelPhase1HitsAnalyzerV.clone()
+pixelOnlyHitsAnalyzerV.tracksTag = cms.InputTag("pixelTracks")
```
Comment (fwyzard) [inline Validation/SiPixelPhase1ConfigV/python/SiPixelPhase1OfflineDQM_sourceV_cff.py:41]:
```suggestion
pixelOnlyHitsAnalyzerV = SiPixelPhase1HitsAnalyzerV.clone(
    tracksTag = 'pixelTracks'
)
```

**Comment by fwyzard** [comment]
I would propose to merge this PR right after #549, which changes the workflow `.501` to run the Patatrack pixel tracks on CPU.
Do you foresee any changes will be needed here once #549 is merged ?

By the way, how can I test that these changes behave as expected?
What plots / etc. should I see ?

**Comment by sroychow** [comment]
> I would propose to merge this PR right after #549, which changes the workflow `.501` to run the Patatrack pixel tracks on CPU.
> Do you foresee any changes will be needed here once #549 is merged ?

No we don't foresee any changes after #549 is merged. 
 
> By the way, how can I test that these changes behave as expected?
> What plots / etc. should I see ?

This PR should not change any existing DQM, Validation plots related to tracks, vertices. Instead it adds new Validation plots for digis, clusters and RecHits(e.g. https://tinyurl.com/y3sfnxy2).

**Pull request merged**